app/src/main/java/com/dcrandroid/dialog/send/SendDialog.kt

```diff
@@ -124,9 +128,84 @@ class SendDialog(val fragmentActivity: FragmentActivity, dismissListener: Dialog
         clearEstimates()
     }
 
+    override fun onPause() {
+        super.onPause()
+        savedInstanceState = Bundle()
+        if (savedInstanceState != null) {
+            //fetch save the selected from wallet ID and selected from account number
+            val selectedSourceAccountID: Long? = amountHelper.selectedAccount?.walletID
+            val selectedSourceAccountNo: Int? = amountHelper.selectedAccount?.accountNumber
+            savedInstanceState?.putLong("selectedSourceAccountID", selectedSourceAccountID!!)
+            savedInstanceState?.putInt("selectedSourceAccountNo", selectedSourceAccountNo!!)
+
+            //fetch and save the selected to wallet ID and selected to account number
+            val selectedDestAccountID: Long? = destinationAddressCard.destinationAccountSpinner.selectedAccount?.walletID
+            val selectedDestAccountNo: Int? = destinationAddressCard.destinationAccountSpinner.selectedAccount?.accountNumber
+            savedInstanceState?.putLong("selectedDestAccountID", selectedDestAccountID!!)
+            savedInstanceState?.putInt("selectedDestAccountNo", selectedDestAccountNo!!)
```

Comment:
`Account` class is already serializable so instead of saving the wallet id and account number, you can save the account object and get the properties from it in `onResume`. Example:

`savedInstanceState?.putSerializable("selectedDestAccount", destinationAddressCard.destinationAccountSpinner.selectedAccount)`